### ISSUE TITLE
"Looking for external crate: gprbuild" is slow

### ISSUE THREAD

**Issue by robdaemon** [alire-project/alire#1028]
When running `alr build`, or any alr command that invokes gprbuild, there is a several second pause while I see "Looking for external crate: gprbuild". Sometimes the pause can be up to 10 seconds.

`alr config` for my project reports:

```
last_build_profile=DEVELOPMENT
toolchain.external.gprbuild=TRUE
toolchain.assistant=false
toolchain.use.gnat=gnat_external=12.1.1
toolchain.use.gprbuild=gprbuild=2020.0.0
toolchain.external.gnat=TRUE
```

`alr version`:

```
APPLICATION                                                                                       
alr version:               1.2.0-rc1                                                              
libalire version:          1.2.0-rc1                                                              
compilation date:          2022-05-18 13:19:00                                                    
compiler version:          Community 2021 (20210519-103)                                          
                          
CONFIGURATION             
config folder:             /home/rroland/.config/alire                                            
force flag:                FALSE                                                                  
non-interactive flag:      FALSE                                                                  
community index branch:    stable-1.2                                                             
compatible index versions: ^1.1 & <=1.2                                                           
indexes folder:            /home/rroland/.config/alire/indexes                                    
indexes metadata:          OK                                                                     
index #1:                  (community) git+https://github.com/alire-project/alire-index#stable-1.2
toolchain assistant:       disabled                                                               
tool #1 gnat:              gnat_external=12.1.1                                                   
tool #2 gprbuild:          gprbuild=2020.0.0                                                      
                          
WORKSPACE                 
root status:               VALID                                                                  
root release:              plaidcl=0.1.0-dev                                                      
root load error:           none                                                                   
root folder:               /home/rroland/src/plaidcl                                              
current folder:            /home/rroland/src/plaidcl                                              
                          
SYSTEM                    
distribution:              FEDORA                                                                 
host-arch:                 X86_64                                                                 
os:                        LINUX                                                                  
target:                    NATIVE                                                                 
toolchain:                 SYSTEM                                                                 
word-size:                 BITS_64                                                                
```

This is Fedora 36 amd64, with the gprbuild and gnat packages installed from the Fedora DNF repos.

**Comment by mosteo** [comment]
Thanks for the report, @robdaemon . I'll take a look at the package manager switches we're using.

As a workaround, when using an external compiler, you should be able to get by in most cases by choosing "None" for both tools.

**Comment by mosteo** [comment]
Fixed in #1036
